### PR TITLE
1894 - untranslatable strings

### LINKED ISSUE #1894: Untranslatable strings
QQ截图20201207113843
QQ截图20201207114011
QQ截图20201207115315
QQ截图20201207122120
Chinese Translation on https://translate.revolutionarygamesstudio.com/ has been finished but there are still some strings remains untranslatable.

### PR DESCRIPTION
**Pull request by zackdrake**
fixes #1894

### CODE REVIEW

File: src/auto-evo/RunResults.cs [hunk 1]
```diff
@@ -222,7 +222,7 @@ void OutputPopulationForPatch(Species species, Patch patch, long population)
                 else
                 {
                     builder.Append("  ");
-                    builder.Append(" went extinct in ");
+                    builder.Append(TranslationServer.Translate("EXTINCT"));
```
Comment (Kasterisk) [inline src/auto-evo/RunResults.cs:225]:
```suggestion
                    builder.Append(TranslationServer.Translate("WENT_EXTINCT_IN"));
```

File: src/auto-evo/RunResults.cs [hunk 2]
```diff
@@ -368,7 +368,7 @@ void OutputPopulationForPatch(Species species, Patch patch, long population)
                 if (GetGlobalPopulation(entry.Species, resolveMoves) <= 0)
                 {
                     builder.Append(" ");
-                    builder.Append("went extinct from the planet");
+                    builder.Append(TranslationServer.Translate("WENT_EXTINCT"));
```
Comment (Kasterisk) [inline src/auto-evo/RunResults.cs:371]:
```suggestion
                    builder.Append(TranslationServer.Translate("WENT_EXTINCT_FROM_PLANET"));
```

**Review by Kasterisk — CHANGES_REQUESTED** [review]

File: src/auto-evo/RunResults.cs [hunk 3]
```diff
@@ -222,7 +222,7 @@ void OutputPopulationForPatch(Species species, Patch patch, long population)
                 else
                 {
                     builder.Append("  ");
-                    builder.Append(" went extinct in ");
+                    builder.Append(TranslationServer.Translate("WENT_EXTINCT_IN"));
                     builder.Append(PatchString(patch));
```
Comment (hhyyrylainen) [inline src/auto-evo/RunResults.cs:226]: For some languages it is not natural to have this say "went extinct in x" so this should instead use a translation string with placeholder in it that uses PatchString(path) as the parameter to format.

File: src/auto-evo/RunResults.cs [hunk 4]
```diff
@@ -222,7 +222,7 @@ void OutputPopulationForPatch(Species species, Patch patch, long population)
                 else
                 {
                     builder.Append("  ");
```
Comment (hhyyrylainen) [inline src/auto-evo/RunResults.cs:224]: This should have one space added, right to reflect that " went extinct in" started with a space.

**Review by hhyyrylainen — CHANGES_REQUESTED** [review]

**Comment by Kasterisk** [comment]
Updated the PR, I think this is fine to merge now.

**Review by Kasterisk — APPROVED** [review]

**Comment by hhyyrylainen** [comment]
I'll re-check this.

**Review by hhyyrylainen — APPROVED** [review]

**Comment by hhyyrylainen** [comment]
@zackdrake we are in the process of getting a CLA in-place for Thrive in order to make it possible to have Thrive on Steam. Could you maybe sign the CLA? You can do that here: https://dev.revolutionarygamesstudio.com/cla

**Pull request merged**